### ISSUE TITLE
Notifications: Digest

### ISSUE THREAD

**Issue by flavour** [sahana/WACOP#5]
Daily Digest for Notifications
Will need some significant work as framework doesn't support this yet

**Comment by devinbalkind** [comment]
Okay I'd imagine we need designs for these email as well? Is there a
standard EDEN email template we should work from?

**Comment by flavour** [comment]
Yes we will, but let's get non-digest notifications all working first before we look at digest notifications?
The Subject is defined here and easy to change:
https://github.com/sahana/eden/blob/master/modules/s3cfg.py#L2266

The body default is here:
https://github.com/sahana/eden/blob/master/modules/s3/s3notify.py#L543
Easy to plugin a custom renderer, so shouldn't be too hard to build to a design of your choice
